services/traefik/docker-compose.yml

```diff
@@ -0,0 +1,64 @@
+version: '3.7'
+
+services:
+  reverse-proxy:
+    # The official v2.0 Traefik docker image
+    image: traefik:v2.0
+    init: true
+    # Enables the web UI and tells Traefik to listen to docker
+    command: --api.insecure=true --providers.docker
+    ports:
+      # The HTTP port
+      - "80:80"
+      # The HTTPS port
+      - "443:443"
+      # The Web UI (enabled by --api.insecure=true)
+      - "8080:8080"
+    volumes:
+      # So that Traefik can listen to the Docker events
+      - /var/run/docker.sock:/var/run/docker.sock
+      - ./dynamic_conf.yml:/etc/traefik/dynamic_conf.yml
+    configs:
+      - source: traefik_config.yml
+        target: /etc/traefik/traefik.yml
```

Comment:
how about having he static configuration for traefik in the docker compose too ? as per my example ?

Reply:
the other stacks are also done similarly so that it's possible to push them through the portainer api.
This way the config files are automatically pushed together with the compose file.